app/models/invenio_rdm_record_converter.rb

```diff
@@ -157,7 +157,7 @@ def invenio_metadata(gf)
       "formats": gf.mime_type,
       "version": version(gf.content),
       "rights": rights(gf.rights),
-      "locations": {"features": gf.based_near.present? ? gf.based_near.shift.split("', ").map{ |location| {place: location.gsub("'", "")} } : []},
+      "locations": {"features": gf.subject_geographic.present? ? gf.subject_geographic.map{ |location| {place: location.gsub("'", "")} } : []},
```

Comment:
Curious: Why do we have to do `location.gsub("'", "")`? (Other characters to remove?)

Reply:
Ah, good catch. We do not need it, I missed removing this.